src/metabase/pulse/render/table.clj

```diff
@@ -41,6 +42,25 @@
 (defn- bar-td-style-numeric []
   (merge (style/font-style) (bar-td-style) {:text-align :right}))
 
+(defn- render-bar-component
+  ([color positive? width-in-pixels]
+   (render-bar-component color positive? width-in-pixels 0))
+  ([color positive? width-in-pixels offset]
+   [:div
+    {:style (style/style
+             (merge
+              {:width            (format "%spx" width-in-pixels)
+               :background-color color
+               :max-height       :10px
+               :height           :10px
+               :margin-top       :3px}
+              (if positive?
+                {:border-radius "0px 2px 2px 0px"}
+                {:border-radius "2px 0px 0px 2px"
```

Comment:
you could move the `:border-radius` key up a level and ditch the call to `merge`, then move the `if` expression:

```clj
(if positive? "0px ..." "2px ...")
```

Reply:
No, because there's the `margin-left` to deal with too.